debian/control

```diff
@@ -8,7 +8,7 @@ Vcs-Git: https://github.com/endlessm/eos-phone-home.git
 
 Package: eos-phone-home
 Architecture: all
-Depends: wget, python, ${misc:Depends}
+Depends: wget, python3, python3-requests, ${misc:Depends}
```

Comment:
Requests buys us nice stuff like an automatic retry loop, but this job gets retried every 3 hours anyway.